subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/ui/SubscriptionWebViewViewModel.kt

```diff
@@ -119,6 +119,7 @@ class SubscriptionWebViewViewModel @Inject constructor(
     fun processJsCallbackMessage(featureName: String, method: String, id: String?, data: JSONObject?) {
         when (method) {
             "backToSettings" -> backToSettings()
+            "backToSettingsActivateSuccess" -> backToSettingsActiveSuccess()
```

Comment:
Do we need to keep handling the `backToSettings` message?

Reply:
That message still works on some screens so yup.